gossip/src/cluster_info_metrics.rs

```diff
@@ -567,6 +568,11 @@ pub(crate) fn submit_gossip_stats(
             stats.require_stake_for_gossip_unknown_stakes.clear(),
             i64
         ),
+        (
+            "save_contact_info_time",
+            stats.save_contact_info_time.clear(),
+            i64
+        ),
```

Comment:
nit: I would put this under `cluster_info_stats2` since there are more timing metrics there and `cluster_info_stats5` is more packet/count related

Reply:
Done